src/entities/hrly-fuel-flow.entity.ts

```diff
@@ -99,6 +107,5 @@ export class HrlyFuelFlow extends BaseEntity {
     c => c.hrlyFuelFlow,
   )
   @JoinColumn({ name: 'hrly_fuel_flow_id' })
-  hrlyParamFuelFlows: HrlyParamFuelFlow[];
-
+  hourlyParameterFuelFlow: HrlyParamFuelFlow[];
```

Comment:
For the other entities, the pattern followed for the names has been the entity name and making it plural.
To be consistent it should be: `hrlyParamFuelFlows: HrlyParamFuelFlow[];`